libs/server/GlobUtils.cs

```diff
@@ -9,11 +9,13 @@ namespace Garnet.server
     public static class GlobUtils
     {
         /// <summary>
-        /// Glob-style pattern matching
+        /// Glob-style ASCII pattern matching
         /// </summary>
         /// <returns>Whether match was found</returns>
-        public static unsafe bool Match(byte* pattern, int patternLen, byte* key, int stringLen, bool nocase = false)
+        public static unsafe bool Match(byte* pattern, int patternLen, byte* key, int stringLen, bool ignoreCase = false)
         {
+            static byte ToLowerAscii(byte value) => (byte)(value | 0x20);
```

Comment:
This code considers @=`, and ^=~ (all other relevant special characters are already handled below). IMO, this behavior is absolutely fine for our use case, but it might be worth adding this limitation to the doc comment for Match().

Reply:
Yeah, I thought about whether to add checks around that but decided it's okay in here, i.e. it is only used for both operands to equality/or in the range pattern.

I'll add the check for upper-case ASCII just in case 👍